frontend/tsconfig.prod.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "./tsconfig",
+  "exclude": ["./src/__tests__/**"]
```

Comment:
Seems like this path change was missed out after the `__tests__` -> `tests` change.

Reply:
@chinmaykunkikar, here I needed your opinion. Shall we keep it as __tests__ even in the backend? or let it be tests?  I guess __tests__ would be better as @shmbajaj mentioned, it would help in separation of concerns

Reply:
I agree with @shmbajaj on this. `__tests__` is a standard practice. And it will definitely stand out of the other code that goes into prod.

Reply:
Lets have it as `__tests__`